frontend/public/locales/en/translation.json

```
@@ -1252,6 +1254,10 @@
   "Enter the application set name": "Enter the application set name",
   "Enter the base DNS domain": "Enter the base DNS domain",
   "Enter the basic credentials information": "Enter the basic credentials information",
+  "Enter the catalog source": "Enter the catalog source",
+  "Enter the catalog source of a submariner subscription, if left blank the default values will be used.": "Enter the catalog source of a submariner subscription, if left blank the default values will be used.",
```

```suggestion
  "Enter the catalog source of a Submariner subscription. If left blank, the default values will be used.": "Enter the catalog source of a Submariner subscription. If left blank, the default values will be used.",
```
This is a comma-splice, so I suggest changing it to 2 sentences. And I would think Submariner should always be capitalized?